astropy/wcs/tests/test_wcs.py

```diff
@@ -30,21 +25,6 @@
 _WCSLIB_VER = LooseVersion(_wcs.__version__)
 
 
-def _check_v71_dateref_warnings(w, nmax=None):
```

Comment:
Given that `wcslib` is bundled, there is no need to check for older versions? With that same reasoning, I also deleted older versions of "validate.txt" files. Please let me know if I misunderstood how the interactions work.

Reply:
We also support using a system version of wcslib. That's why the checks for older versions.

Reply:
Ah, okay. I'll revert those changes. Thanks for the clarifications!